Tests/SDK/Service/HttpRequestRunnerTest.swift

```diff
@@ -0,0 +1,45 @@
+@testable import CIO
+import Foundation
+#if canImport(FoundationNetworking)
+import FoundationNetworking
+#endif
+import XCTest
+
+/**
+ Test that performs a real HTTP request to Customer.io API. This is a convenient way to test the
+ networking layer of the SDK. These tests are meant to run on your local machine, not CI server.
+
+ Setup:
+ 1. In XCode, go to: Edit Scheme > Run
+ 2. Create 2 environment variables: `SITE_ID` and `API_KEY`. Populate those values with random credentials from a Workspace.
```

Comment:
> Populate these with a set of testing credentials from your workspace.

New creds are cheap, and would prefer customers get in the habit of creating ones dedicated to their use case that they can then more easily deprovision

Reply:
I wonder if there is a miscommunication? 

The instructions here are for internal SDK developers, only. Setting up XCode to run these tests for you. Not for customers to install or use the SDK. 

The instructions are asking you (the internal SDK developer) to go into a workspace that you use for testing and use those credentials.

Reply:
gotcha. I'd still avoid the word `random` and do something like
```
a set of test credentials from a workspace you control
```
or something to make it clearer

Reply:
Great idea. Done.